iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -826,6 +826,51 @@ LogicalResult ReverseOp::generateScalarImplementation(OpBuilder &b,
   return success();
 }
 
+Operation *ReverseOp::getTiledImplementation(OpBuilder &builder,
+                                             ValueRange outputs,
+                                             ArrayRef<OpFoldResult> offsets,
+                                             ArrayRef<OpFoldResult> sizes,
+                                             SmallVectorImpl<Value> &results) {
+  int64_t rank = getOperandRank();
+  SmallVector<OpFoldResult> strides(rank, builder.getI64IntegerAttr(1));
+  Location loc = getLoc();
+  SmallVector<Value> tiledOperands;
+  tiledOperands.emplace_back(
+      getSlice(builder, loc, input(), offsets, sizes, strides));
+
+  SmallVector<OpFoldResult> mirrorOffsets(offsets.begin(), offsets.end());
+  for (auto dim : dims()) {
+    auto size = getDimValue(builder, loc, input(), dim);
+    Value mirrorOffset =
+        getValueOrCreateConstantIndexOp(builder, loc, mirrorOffsets[dim]);
+    mirrorOffset = builder.create<SubIOp>(loc, size, mirrorOffset);
```

Comment:
Could we use `affine_map` to express this index computation. They canonicalize better

Reply:
Good point, done!